core/templates/pages/topic-viewer-page/topic-viewer-page.component.css

```diff
@@ -110,8 +110,7 @@
 }
 
 .topic-viewer-page-content .tab-title {
-  color: #01645c;
-  font-family: 'Capriola', 'Roboto', Arial, sans-serif;
```

Comment:
Why are we changing the color and fonts in this PR?

Reply:
oppia.org contains the ideal behavior of text's style of heading (place values). By ideal, I mean as described in Om's doc. (https://docs.google.com/document/d/1-bRcvAM5bUhuohK5xovu4K13qU40xycmnAeQr6_45Pc/edit#heading=h.c91sh8euhoqu)